src/main/java/com/ericsson/eiffel/remrem/publish/config/SpringLoggingInitializer.java

```diff
@@ -25,10 +25,10 @@ public void initialize(ConfigurableApplicationContext applicationContext) {
 		Class[] loggers = {SpringApplication.class, App.class, ConfigFileApplicationListener.class, EndpointMBeanExporter.class,
 				AutoConfigurationReportLoggingInitializer.class};
 		Logger log = (Logger) LoggerFactory.getLogger("ROOT");
-		log.setLevel(Level.ERROR);
+		log.setLevel(Level.INFO);
 		for (Class logger : loggers) {
 			log = (Logger) LoggerFactory.getLogger(logger);
-			log.setLevel(Level.ERROR);
+			log.setLevel(Level.INFO);
```

Comment:
log level should also be error so that in production we only have have errors visible for users. You can maybe add an argumet for logging level in yaml properties and as argument from CLI